CoreFoundation/Base.subproj/CFKnownLocations.c

```diff
@@ -72,6 +72,25 @@ CFURLRef _Nullable _CFKnownLocationCreatePreferencesURLForUser(CFKnownLocationUs
         }
     }
     
+#elif DEPLOYMENT_TARGET_WINDOWS
+
+    switch (user) {
+        case _kCFKnownLocationUserAny:
+            location = CFURLCreateWithFileSystemPath(kCFAllocatorSystemDefault, CFSTR("\\Users\\All Users\\AppData\\Local"), kCFURLWindowsPathStyle, true);
```

Comment:
Isn’t there API to get this path, and the user path below?

Reply:
No, the path is just "\Users\All Users\AppData\Local" ... the last component is a decision (Local vs LocalLow vs Roaming).  The user name substitution is usually a `_snprintf` type of thing